GRAPHICS/SetParticleFxNonLoopedColour.md

```diff
@@ -8,10 +8,13 @@ ns: GRAPHICS
 void SET_PARTICLE_FX_NON_LOOPED_COLOUR(float r, float g, float b);
 ```
 
-only works on some fx's, not networked
-
 ## Parameters
-* **r**: 
-* **g**: 
-* **b**: 
+* **r**: Red (0.0 - 1.0)
+* **g**: Green (0.0 - 1.0)
+* **b**: Blue (0.0 - 1.0)
+
+Only works on some fx's, while on others it might SEEM to work "properly", but the colors can be "strange" or even completly different from what you've expected. Reason for this is that those fx's might already have colors "baked into them" which then start to act as a "mixing palette", resulting in a different color than expected. A hypothetical example of this would be if the fx itself is already full (bright) red (RGB: 1.0, 0.0, 0.0) and you then set the color to (bright) green (RGB: 0.0, 1.0, 0.0), that it MIGHT result in Yellow (RGB: 1.0, 1.0, 0.0).
```

Comment:
Please move these descriptions above the Parameters section, otherwise it won't work correctly.

Reply:
If anyone else wants to take over (or re-submit), be my guest, the code / call works but seems to require additional testing requested by CFX. I personally have no interest in developing stuff for FiveM anymore, for several reasons which don't matter here.

Of someone wants to takes 'request' and implement then, go a head, otherwise i will in about a month or so will retract them and remove from my own account (close the pull request).